src/include/distributed/metadata_cache.h

```diff
@@ -134,6 +134,8 @@ typedef enum
 	ANY_CITUS_TABLE_TYPE
 } CitusTableType;
 
+
+extern bool IsLocalOrCitusLocalTable(Oid relationId);
```

Comment:
what about mat views? They can be considered as local tables as well
```SQL 

CREATE MATERIALIZED VIEW mat_view AS SELECT * FROM local;
SELECT count(*) FROM distributed JOIN mat_view USING (id);
ERROR:  relation mat_view is not distributed
Time: 4.901 ms

```

I guess we can use `IsLocalTableRteOrMatView ()`